python/servo/command_base.py

```diff
@@ -51,6 +51,10 @@ class CommandBase(object):
     def __init__(self, context):
         self.context = context
 
+        def resolverelative(category, key):
+            if self.config[category][key].startswith("./"):
```

Comment:
`path.join` already ignore arguments before one that is an absolute path: `path.join('/foo', 'bar') == '/foo/bar', path.join('/foo', '/bar') == '/bar'`. Why special-casing `./`?